src/Swarm/TUI/Attr.hs

```diff
@@ -132,6 +132,31 @@ sepAttr = "sep"
 infoAttr = "info"
 defAttr = "def"
 
+worldAttributes :: [AttrName]
```

Comment:
One way to do it is to not allow accessing world attribute names and create a `[(AttrName, Attr)]` list of world attributes.

Those I could filter and pass directly to the attribute map. :smiling_imp:

Reply:
Yeah, we really don't need Haskell constants, I think we should get rid of them.  We can easily look up attributes by name from some kind of map or association list.

Reply:
Well, I meant keeping constants _only_ for those we use in the Haskell code and _**not** those used in the world map*_.

<sup>*) With the exception of the few attributes that we use for drawing the logo.</sup>

It would not be great to have outdated attribute strings in code, so I think it is a good idea to keep those. :slightly_smiling_face:

Reply:
@byorgey I just pushed my solution, what do you think? :slightly_smiling_face: